Library/Homebrew/livecheck/strategy/github_latest.rb

```diff
@@ -41,24 +41,13 @@ class GithubLatest
         # `strategy :github_latest` in a `livecheck` block.
         PRIORITY = 0
 
-        # The `Regexp` used to determine if the strategy applies to the URL.
-        URL_MATCH_REGEX = %r{
-          ^https?://github\.com
-          /(?:downloads/)?(?<username>[^/]+) # The GitHub username
-          /(?<repository>[^/]+)              # The GitHub repository name
-        }ix.freeze
-
-        # The default regex used to identify a version from a tag when a regex
-        # isn't provided.
-        DEFAULT_REGEX = %r{href=.*?/tag/v?(\d+(?:\.\d+)+)["' >]}i.freeze
-
         # Whether the strategy can be applied to the provided URL.
         #
         # @param url [String] the URL to match against
         # @return [Boolean]
         sig { params(url: String).returns(T::Boolean) }
         def self.match?(url)
-          URL_MATCH_REGEX.match?(url)
+          GithubRelease::URL_MATCH_REGEX.match?(url)
```

Comment:
```suggestion
          GitHubRelease::URL_MATCH_REGEX.match?(url)
```

and elsewhere if change below is made